pkgs/development/tools/ocaml/ocamlformat/generic.nix

```diff
@@ -39,7 +41,23 @@ buildDunePackage rec {
   useDune2 = true;
 
   buildInputs =
-    if lib.versionAtLeast version "0.14"
+    if lib.versionAtLeast version "0.15.1"
```

Comment:
Any special reason for having 0.15.0 and 0.15.1?

Reply:
The only difference between 0.15.0 and 0.15.1 is the change in dependency (ocaml-migrate-parsetree vs ppxlib) but the exact version should be specified and checked into projects that use this tool. This means that every versions of OCamlformat are required.

Reply:
Alright fair, bit annoying that this means we specify a few not strictly required buildInputs for 0.15.1, but doesn't matter too much I think.